taigit/src/actions/taigaActions.js

```diff
@@ -28,3 +29,13 @@ export const grabSprintStats = (infoForApiCall) => dispatch => {
       dispatch({type: GET_SPRINT_STATS, payload: sprintStats});
     });
 }
+
+export const grabSingleSprintData = (infoForApiCall) => dispatch => {
+  //get_task_details(sprint_id: number, project_id: any, sprint_name: string)  : Promise<Object> {
+    console.log("GRAB SPRINT DATA");
+    get_task_details(220752, 306316,'Sprint 2 - Taiga')
```

Comment:
I would add these as parameters that get passed in like _infoForApiCall_.

Reply:
ohh! Sure.. I will do.